Dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM golang:1.19
+
+RUN echo "deb http://security.debian.org/debian-security stretch/updates main" >> /etc/apt/sources.list
+
+RUN apt update 
+RUN apt install -y ffmpeg
+
+ENV GO111MODULE=on \
+    CGO_ENABLED=0 \
+    GOOS=linux \
+    GOARCH=amd64
+
+WORKDIR /build
+
+#COPY go.mod .
```

Comment:
why these are comments?

Reply:
i can remove that, basically this commented code was initially used for creating and testing  the image for zestream server